### DISCUSSION TITLE
ImportError: cannot import name 'readsou_tu_data'

### DISCUSSION THREAD

**Discussion by leesoohong** [pyg-team/pytorch_geometric#3658] [Q&A]
I have installed all the necessary packages by following the guidelines on the website. 

pip install torch-scatter -f https://data.pyg.org/whl/torch-1.8.0+cu101.html
pip install torch-sparse -f https://data.pyg.org/whl/torch-1.8.0+cu101.html
pip install torch-geometric
pip install torch-cluster -f https://data.pyg.org/whl/torch-1.8.0+cu101.html
pip install torch-spline-conv -f https://data.pyg.org/whl/torch-1.8.0+cu101.html

But I am getting the following error, can you help me?
**ImportError: cannot import name 'readsou_tu_data' from 'torch_geometric.io'**

**Comment by rusty1s** [comment]
Do you mean `read_tu_data`? Can you show me the complete traceback?

In addition, for older PyTorch versions, it might be necessary to specify the versions of packages, see [here](https://data.pyg.org/whl/torch-1.8.0+cu101.html)
```
pip install torch-scatter==2.0.8 torch-sparse==0.6.12 torch-cluster==1.5.9 torch-spline-conv==1.2.1 -f https://data.pyg.org/whl/torch-1.8.0+cu101.html
```
